scratch-gui/src/components/library/library.css

```
@@ -9,56 +16,62 @@
 
 .filter-bar {
     display: flex;
     flex-direction: row;
     justify-content: flex-start;
-    align-items: center;
-    height: calc($library-filter-bar-height + 2rem); /* padding */
-    background-color: $looks-transparent;
-    padding: 0 1rem;
-    font-size: .875rem;
+    width: 30px;
+    border: 0;
```

Stylelint (17.12.0)

[error] Duplicate property "border" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)